libs/domain/picking/src/components/customer-note/src/customer-note.component.ts

```diff
@@ -41,8 +41,14 @@ export class CustomerNoteComponent extends PickingListMixin(LitElement) {
 
       <p>${this.pickingList?.cartNote}</p>
 
-      <oryx-button>
-        <button @click=${this.onProceed}>
+      <oryx-button
+        ?loading=${this.upcomingPickingListId === this.pickingList.id}
+      >
+        <button
```

Comment:
`?disabled=${!!this.upcomingPickingListId}`

Reply:
loading state makes the button inactive, so the user can't interact with it.